tests/graph-codegen.test.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs')
+const { join } = require('path')
+
+const test = require('ava')
+const { buildSchema, parse } = require('graphql')
+
+const {
+  extractFunctionsFromOperationDoc,
+  generateFunctionsSource,
+  generateHandlerSource,
+} = require('../src/lib/one-graph/netlify-graph')
+
+const { normalize } = require('./utils/snapshots')
+
+const netligraphConfig = {
+  extension: 'mjs',
```

Comment:
mjs for a cjs file

Reply:
Is this wrong? I've been asking about this for a long time, I have no idea:
1. What the correct conventions are
2. How to autodetect them

At this point, I figured I'd get this merged, start getting complaints from people, and then fix it incrementally.

Reply:
"mjs" is for ESM